[2025-07-07T19:46:07.072+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_activity_analysis.transform_data manual__2025-07-07T19:45:56.179762+00:00 [queued]>
[2025-07-07T19:46:07.079+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_activity_analysis.transform_data manual__2025-07-07T19:45:56.179762+00:00 [queued]>
[2025-07-07T19:46:07.079+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-07-07T19:46:07.093+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): transform_data> on 2025-07-07 19:45:56.179762+00:00
[2025-07-07T19:46:07.103+0000] {standard_task_runner.py:57} INFO - Started process 1551 to run task
[2025-07-07T19:46:07.105+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'customer_activity_analysis', 'transform_data', 'manual__2025-07-07T19:45:56.179762+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/customer_activity_dag.py', '--cfg-path', '/tmp/tmprbr7fhgg']
[2025-07-07T19:46:07.106+0000] {standard_task_runner.py:85} INFO - Job 50: Subtask transform_data
[2025-07-07T19:46:07.120+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-07-07T19:46:07.143+0000] {task_command.py:410} INFO - Running <TaskInstance: customer_activity_analysis.transform_data manual__2025-07-07T19:45:56.179762+00:00 [running]> on host 7a17032e36f4
[2025-07-07T19:46:07.433+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ds_const@mail.ru' AIRFLOW_CTX_DAG_OWNER='Daniil' AIRFLOW_CTX_DAG_ID='customer_activity_analysis' AIRFLOW_CTX_TASK_ID='transform_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-07T19:45:56.179762+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-07T19:45:56.179762+00:00'
[2025-07-07T19:46:07.452+0000] {logging_mixin.py:150} INFO - /opt/***/shared_tmp/extract_manual__2025-07-07T19:45:56.179762+00:00.csv
[2025-07-07T19:46:08.676+0000] {logging_mixin.py:150} INFO - Файл /opt/***/shared_tmp/extract_manual__2025-07-07T19:45:56.179762+00:00.csv успешно удалён
[2025-07-07T19:46:08.676+0000] {logging_mixin.py:150} INFO - Received date type: <class 'datetime.datetime'>, value: 2024-04-01 00:00:00
[2025-07-07T19:46:09.140+0000] {logging_mixin.py:150} WARNING - Generating flags:   0%|          | 0/10 [00:00<?, ?it/s]
[2025-07-07T19:46:09.154+0000] {logging_mixin.py:150} WARNING - Generating flags: 100%|##########| 10/10 [00:00<00:00, 756.19it/s]
[2025-07-07T19:46:09.499+0000] {python.py:183} INFO - Done. Returned value was: /opt/***/shared_tmp/transform_manual__2025-07-07T19:45:56.179762+00:00.csv
[2025-07-07T19:46:09.524+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=customer_activity_analysis, task_id=transform_data, execution_date=20250707T194556, start_date=20250707T194607, end_date=20250707T194609
[2025-07-07T19:46:09.563+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-07-07T19:46:09.582+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
